duplicates-in-array.js

```diff
@@ -0,0 +1,23 @@
+module.exports = function (array, n) {
+
+    var length = array.length;
+    var counter = zeroFilledArray(array.length);
+
+    if (array.length === 0) return false;
+    if (n != length) return false;
+    for (var index = 0; index < array.length; index++) {
+      if (array[index] < 1 || array[index] > n) {
+          return false;
+      } else {
+    		if(counter[array[index]] === 1){
```

Comment:
Can you think of a method to not use the bit array to verify for duplicates?

Reply:
At first I did this problem by verifying if the sum of all numbers is n(n+1)/2.  If it wasn't equal then I had duplicates, but then I saw that this case didn't work If I did not have the array as a series from 1 to n.  I will try to do it in another way.